actionRouter.js

```diff
@@ -0,0 +1,108 @@
+const express = require('express');
+
+const router = express.Router();
+
+const actionDb = require('./data/helpers/actionModel');
+const projectDb = require('./data/helpers/projectModel');
+
+//GET ALL ACTIONS
+router.get('/', (req, res) => {
+    actionDb.get()
+    .then(actions => {
+        res.status(200).json(actions)
+    })
+    .catch(err => {
+        res.status(500).json({ error: "GET FAILED!"})
+    })
+});
+
+
+//GET ACTION BY ID
+router.get('/:id', validateAction, (req, res) => {
+    actionDb.get(req.action)
+    .then(action => {
+        res.status(200).json(action)
+    })
+    .catch(err => {
+        res.status(500).json({ error: "GET ACTION FAILED"})
+    })
+});
+
+
+//CREATES NEW ACTION AFTER VALIDATING ID
+router.post('/', validateProject, (req, res) => {
+    const {description, notes} = req.body;
+    req.body.project_id = req.project;
```

Comment:
**From our 1:1 -**

In this section of code, `req.project` is undefined. Do you remember why?